qasp/agile_bpa_qasp.md

```diff
@@ -0,0 +1,106 @@
+for the final deliverable. However, the sprints and incremental delivery of code
+will be assessed by the Government to ensure that the contractor is on a path to
+successful final delivery.
+
+Deliverable or Required Services Performance Standard(s):
+
+  **Deliverable or Required Services** | **Performance Standard(s)** | **Acceptable Quality Level** | **Method of Surveillance**
+   ----------- | ---------------- | -------------- | ---------------
+   Tested Code | Code delivered under the order must have substantial test code coverage and a clean code base. | Minimum of 90% test coverage of all relevant code | Combination of manual review and automated testing, using agreed-upon publicly-available SaaS products
+   Accessible | Client-side rendering must conform with section 508 standards and WCAG2AA standards. | 0 errors reported for WCAG2AA using an automated scanner, 0 errors reported in manual testing, and all warnings confirmed as non-issues |  http://squizlabs.github.io/HTML_CodeSniffer/ or https://github.com/pa11y/pa11y
+   Deployed | Code must successfully build and deploy into staging environment. | Successful build with a single command | Combination of manual review and automatic testing
+   Documented | All dependencies (and licenses for dependencies) are listed and all major functions are documented. | All dependencies are listed and the licenses are documented. Software/source code is documented. System diagram is provided. | Combination of manual review and automatic testing
+   Available | Code must be stored in a version-controlled open-source repository. | All of the code needed to run project must be publicly available. | TTS will assess code availability by installing and running the project on a fresh machine.
+   User research | Usability testing and other user research methods must be conducted at regular intervals throughout the development process (not just at the beginning or end). |  Artifacts from usability testing and/or other research methods with end-users are available at the end of every applicable sprint, in accordance with the vendor’s research plan. | TTS will evaluate the artifacts based on a research plan provided by the vendor at the end of the second sprint.
+   Secure | Code must be free of medium- and high-level static and dynamic security vulnerabilities | Clean tests from a static testing SaaS, such as Gemnasium, and from OWASP ZAP, and/or documentation explaining any false positives. |  https://pages.18f.gov/before-you-ship/
+
+PROCEDURES
+----------
+
+The COR/ACOR, along with the TTS Product Lead and the product owner, will
+inspect all tasks required by the call order to ensure contractor compliance
+with the call order requirements at the conclusion of each sprint, which shall
+have a length of two weeks or less. Delivery of the code will occur by pull
+request from the contractor’s repository to the TTS repository. If inspection
```

Comment:
Yeah...I don't know what to do about this. Maybe "TTS-owned repository"? I was hoping to avoid saying TTS and 18F, but that might not be possible.